data/formats-data.js

```diff
@@ -3799,7 +3806,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	shieldon: {
-		randomBattleMoves: ["stealthrock", "metalburst", "fireblast", "icebeam", "protect", "toxic", "roar"],
+		randomBattleMoves: ["stealthrock", "metalburst", "fireblast", "icebeam", "protect", "toxic", "roar", "heavyslam", "rockblast"],
```

Comment:
["stealthrock", "metalburst", "protect", "toxic", "roar", "heavyslam", "rockblast"]

Reply:
gotta keep at least one special move or the drabness of its move colors will make me kms
in seriousness i can see how they could get in the way but its harder to rationalize it when things like toxictect are what theyd get in the way of...lets at least keep fire blast so we don't build the vacation home of every steel type

Reply:
hm... you'd rather switch out tbh lol. (not like you can switch out against magnemite, and aron uses eq/superpower)

Reply:
ideally you'd first do it in a switch, perhaps a switch facilitated by Steels earlier coming in as your utilitymoves give them confidence